tests/test_single_topology_confgen.py

```diff
@@ -139,3 +139,41 @@ def test_confgen_spot_edges():
         mol_a = get_mol_by_name(mols, src)
         mol_b = get_mol_by_name(mols, dst)
         run_edge(mol_a, mol_b, protein_path, n_windows)
+
+
+@pytest.mark.parametrize("pair", [("35", "84")])
+@pytest.mark.parametrize("n_windows", [2])
+@pytest.mark.parametrize("seed", [2023])
+def test_min_cutoff_failure(pair, seed, n_windows):
+    """Verify that when minimizing within solvent only the ligand is considered for movement of atoms
+    during minimization. Expectation is that all atoms of the molecules will move beyond effectively zero cutoff"""
+    src, dst = pair
+    box_width = 4.0
+    min_cutoff = 1e-8
+
+    ligands = "timemachine/datasets/fep_benchmark/hif2a/ligands.sdf"
```

Comment:
Assuming that this is a specific example where non-ligand atoms move more than the cutoff? (might call this out to clarify)

Reply:
Sorry, never mind this. I see now that you're just setting the cutoff really small so that every atom is probably moving more than the cutoff, and asserting that only the ligand atoms are triggering the failure.

Reply:
Added comment in https://github.com/proteneer/timemachine/pull/1336/commits/5fc2486e48a1b8ab644e2cbbbc0d3202c73fbb50